.github/workflows/pull-request-ci.yaml

```diff
@@ -80,6 +80,66 @@ jobs:
           name: kubecf-bundle.tgz
           path: ${{ github.workspace }}/bundle
 
+  build-next:
```

Comment:
I would have just made `build` use a matrix instead, but sure. (We can explicitly set `VERSION` to empty string to keep the existing behaviour.)

Reply:
True, that's a better approach. I take note for following PRs.